### ISSUE TITLE
REPL immediately exits

### ISSUE THREAD

**Issue by csdrane** [Frege/frege-repl#30]
```
frege-repl [master] $ ./gradlew --console plain :frege-repl-core:run
:frege-repl-nativedeps:compileFrege UP-TO-DATE
:frege-repl-nativedeps:compileJava UP-TO-DATE
:frege-repl-nativedeps:compileGroovy UP-TO-DATE
:frege-repl-nativedeps:processResources UP-TO-DATE
:frege-repl-nativedeps:classes UP-TO-DATE
:frege-repl-nativedeps:jar UP-TO-DATE
:frege-repl-core:compileJava UP-TO-DATE
:frege-repl-core:compileFrege UP-TO-DATE
:frege-repl-core:compileGroovy UP-TO-DATE
:frege-repl-core:processResources UP-TO-DATE
:frege-repl-core:classes UP-TO-DATE
:frege-repl-core:run
Welcome to Frege 3.22.524-gcc99d7e (Oracle Corporation Java HotSpot(TM) 64-Bit Server VM, 1.8.0_45)

frege>
BUILD SUCCESSFUL

Total time: 3.514 secs

This build could be faster, please consider using the Gradle Daemon: http://gradle.org/docs/2.5/userguide/gradle_daemon.html
frege-repl [master] $
```

I return to bash immediately. Am I doing something incorrectly?

Unrelated, I think the instructions under `How to run?` are out of date, as the zip file I found did not contain binaries.

**Comment by mmhelloworld** [comment]
Hi,

It is a known issue. I have already fixed it but I haven't pushed my changes yet. Sorry about that! I will be pushing all my changes and provide a new release end of day today (EST). The instructions haven't been updated after we moved to gradle recently. I will fix that as well today.

Meanwhile you could download the prebuilt binaries from releases to run the REPL which is for the previous version of Frege or you may find the online REPL useful http:://try.frege-lang.org .

**Comment by mmhelloworld** [comment]
Sorry, the online REPL URL is incorrect above. on my phone now. The correct URL: http://try.frege-lang.org/

**Comment by mmhelloworld** [comment]
This should be fixed now. I have released a new version: [1.2.1-SNAPSHOT](https://github.com/Frege/frege-repl/releases/tag/1.2.1-SNAPSHOT) and I have updated the instructions as well.
